erpnext_documentation/www/docs/v13/user/manual/en/human-resources/employee-grievance.md

```diff
@@ -0,0 +1,41 @@
+<!-- add-breadcrumbs -->
+# Employee Grievance
+
+> Introduced in version-13
+
+**An employee grievance is a concern, problem, or complaint raised by an employee with what he expects from the company(or their individuals) and its management. Something has made them feel dissatisfied, and they believe it is unfair and/or unjust to them. It may or may not be justified but needs to be tackled very carefully.**
+
+In ERPNext, you can handle these Employee Grievances to make your employees happy.
+
+To access Employee Grievance, go to:
+
+> Home > Human Resources > Employee Grievance
+
+## 1. Prerequisites
+
+Before creating an Attendance record, it is advised that you create the following first:
+
+* [Employee](/docs/user/manual/en/human-resources/employee)
+* [Grievance Type](/docs/user/manual/en/human-resources/grievance-type)
+
+## 2. How to create Employee Grievance
+
+1. Go to Employee Grievance List, click on Add Employee Grievance.
+1. Select Raised by, date, etc.
+1. Fill Grievance Details like Grievance Against Party, Grievance Against, Grievance Type, etc.
+1. Enter Description.
+1. Save your document.
+
+<img class="screenshot" alt="Employee Grievance" src="{{docs_base_url}}/v13/assets/img/human-resources/employee-grievance.png">
+
+> Note: Employee can also link Documents related to the grievance.
+
+
+## 3. Workflow
+
+1. The Employee creates an Employee Grievance document for their complaint or problem against Company, Employee, Department, etc.
+1. The HR manager will investigate and he will fill the cause of the grievance and after that, he can change the status to Investigated. You can not set the status to **Investigated** without cause of grievance.
+1. After that, the Hr manager will fill in resolution detail, and now he can and save the Employee grievance. You can not mark Grievance as **Resolved** without Resolution details.
+1. So, Now the cause of grievance and Resolution details are filled and the status is resolved which allows you to submit the document.
```

Comment:
Once the cause of Grievance and Resolution details are filled and the status is set as Resolved the document can be submitted.

Reply:
done